### ISSUE TITLE
Confused for result with `MA Soft Prototype Probs ` strategy

### ISSUE THREAD

**Issue by Yuxiang1990** [hbzju/PiCO#5]
Hi, 
very nice work. in paper said,  for PICO ablation study,  "However,
**MA Soft Prototype Probs** displays degenerated performance, suggesting soft label assignment is less reliable in identifying the ground-truth", I am confused that with soft prototype probs used for label disambiguation already achieve 91.6, MA Soft Prototype Probs only 81.6%?

**Comment by hbzju** [comment]
Thanks for your attention to our work. In effect, MA hard prototype, which is adopted by PiCO is a relatively aggressive strategy for label disambiguation. We postulate that the MA soft prototype strategy may be too passive (even compared to the soft version) and the model tends to overfit nonconfidential labels at the early stage of training and thus finally diverges. Hope it helps.

**Comment by Yuxiang1990** [comment]
MA soft prototype strategy wasn't adopted based on warm-up strategy? Without warm-up procedure the degenerated result is within the expected range.

**Comment by hbzju** [comment]
We apply the same warm-up strategy as the original PiCO algorithm.
